test/not_loaded_script_test.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
@kevmoo Sorry, I panicked because of my bug with tests in Webstorm :/
BUT, I am solving the thing that the test will not pass on other platforms than Dartium. Because I am testing there the instance of FirebaseJsNotLoadedException via throwsA(new isInstanceOf<fb.FirebaseJsNotLoadedException>()) but compiled to JS is ReferenceError and the test will not pass.

Is there any smarter solution than test only "throws"?